hooks/notifications.hooks.ts

```diff
@@ -1,44 +1,85 @@
 import * as Notifications from "expo-notifications";
+import * as Device from 'expo-device';
+import Constants from 'expo-constants';
+import AsyncStorage from "@react-native-async-storage/async-storage";
```

Comment:
Still a mix of `'` vs `"` floating around.  Try installing prettier, and configuring VSCode to auto-format on save.

Reply:
I have a task for this and I will do it separate from this PR,  on the next update